minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -146,29 +142,29 @@ public boolean isServing() {
   }
 
   /**
-   * Stops the master and cleans up client connections.
+   * Stops the master processes and cleans up client connections.
    */
   public void stop() throws Exception {
-    // This shutdown needs to be done in a loop with retry because the interrupt signal can
-    // sometimes be ignored in the master implementation. For example, if the master is doing
-    // a hdfs listStatus RPC (hadoop version is 1.x), the interrupt signal is not properly handled.
-    while (mMasterThread.isAlive()) {
```

Comment:
Is it safe to get rid of the looping? Calling `interrupt()` may be completely ignored by the master thread. Perhaps we could use the loop, but add a 10 second timeout.